filemanager/xml_templates/baseplate/build_upload.xml

```diff
@@ -3,10 +3,10 @@
 	<PARTS>
 		<PART mode="auto">
 			<KIND_OF_PART>{{ kind_of_part }}</KIND_OF_PART>
-			<RECORD_INSERTION_USER>{{ record_insertion_user }}</RECORD_INSERTION_USER>
+			<!-- <RECORD_INSERTION_USER>{{ record_insertion_user }}</RECORD_INSERTION_USER> -->
 			<SERIAL_NUMBER>{{ ID }}</SERIAL_NUMBER>
 			<LOCATION>{{ location }}</LOCATION>
-			<MANUFACTURER>{{ manufacturer }}</MANUFACTURER>
+			<!-- <MANUFACTURER>{{ manufacturer }}</MANUFACTURER> -->
```

Comment:
These might not be important for the MACs since all the parts (baseplate, sensor and hexaboard) data will come from the central DB instead of being created by the MACs.

I agree that deleting the commented lines would make the XML files neater. But the XML templates are not the final version, as the DB tables are still under development. We will need to update the XML files accordingly.

Reply:
ok fine, so let's just merge this as it is!